invenio_communities/templates/semantic-ui/invenio_communities/details/home/index.html

```diff
@@ -0,0 +1,91 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2023 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_communities/details/base.html" %}
+{%- set title = community.metadata.title -%}
+{% set active_community_header_menu_item= 'home' %}
+
+{%- block page_body %}
+  {{ super() }}
+  <div class="ui rel-m-2 rel-pt-1">
+    {{hero_section(heading=community.theme.heading,subheading=community.theme.subheading)}}
+    {{how_it_works()}}
+  </div>
+{%- endblock page_body -%}
+
+
+{% macro hero_section(heading="Not found", subheading="Not found") %}
+<div class="ui centered stackable theme-font grid container rel-mt-2 rel-mb-4">
+  <div class="thirteen wide column">
+    <div class="row item">
+      <h1 class="ui large header">{{heading}}</h1>
+      <h1 class="ui text-muted small header">{{subheading}}</h1>
+    </div>
+    <div class="ui rel-mt-2 rel-mb-4">
+      <div class="ui fluid container row rel-mb-2">
+          <a href="/uploads/new?community={{ community.slug }}"
+          target="_blank"
+          class="ui positive button labeled theme-secondary icon rel-mt-1">
+         <i class="upload icon" aria-hidden="true"></i>
+         {{ _("New upload") }}
```

Comment:
minor: we had some tickets on Zenodo since the `New upload` seems like you are uploading one record without the community part. Does it make sense to have a different label here?